protocol/xmrtaker/claim.go

```diff
@@ -113,44 +118,49 @@ func (s *swapState) claimMonero(skB *mcrypto.PrivateSpendKey) (mcrypto.Address,
 	}
 
 	log.Infof("monero claimed in account %s; transferring to original account %s",
-		addr, depositAddr)
+		abAddr, depositAddr)
 
 	err = mcrypto.ValidateAddress(string(depositAddr), s.Env())
 	if err != nil {
-		log.Errorf("failed to transfer to original account, address %s is invalid", addr)
-		return addr, nil
+		log.Errorf("failed to transfer to original account, address %s is invalid", abAddr)
+		return abAddr, nil
```

Comment:
wait why do we just log here instead of returning an error?

Reply:
I'll update it to return the error.